scripts/ansible/roles/linux/jenkins/tasks/packer.yml

```diff
@@ -36,6 +36,19 @@
     groups: docker
     append: yes
 
+- name: Determine if sgx_prv group exists
+  getent:
+    database: group
+    key: sgx_prv
+
+- name: Add user to sgx_prv group
```

Comment:
Should add comment that this step is often moot and needs to also be implemented during Jenkins provisioning